prometheus/openshift/files/templates/metrics.yml

```diff
@@ -0,0 +1,11324 @@
+---
```

Comment:
Can we please break this big file up into smaller chunks? Seems to me that there are some good re-usable components in there, but those aren't easy to get to as-is since it's all in one big file.